### ISSUE TITLE
Big is not a function

### ISSUE THREAD

**Issue by mitra42** [MikeMcl/big.js#95]
I'm seeing a new bug 
```
const Big = require('big.js')
...
Big(0)
```
fails with Big is not a function. 
This is in a webpacked environment, it appears Big.Big(0) works, which combined with an update 10 hours ago that says it changed the exports, suggests the exports are not working with webpack.

**Comment by mitra42** [comment]
I can confirm that using package.json to go back to 5.0.3 works.

**Comment by mitra42** [comment]
I'm not totally sure, I'm importing ipfs which imports ipfs-bitswap which is doing the `require('big.js')`.  

I don't know why webpack gets the big.mjs file (we are using `webpack --mode development` which does not itself minify).  The problem seems to be that "const Big = require('big.js')" seems to set Big as a structure of some kind so  `Big(0)` doesn't work.

**Comment by rahulgi** [comment]
Also using webpack. I'm importing big using `import Big from "big.js"`. In the latest version, this gives the error `Unhandled Rejection (TypeError): __WEBPACK_IMPORTED_MODULE_0_big_js___default.a is not a constructor` when trying to do `new Big(0)`. As @mitra42 said above, downgrading to 5.0.3 fixes the issue.

**Comment by MikeMcl** [comment]
Okay guys, let me know if the version I've just published works, *v5.1.1*. 

If not, the problem must be with the change I made to the *package.json*, and I'll probably roll it back.

**Comment by rahulgi** [comment]
giving it a shot!

**Comment by rahulgi** [comment]
@MikeMcl still seeing the issue in v5.1.1.

**Comment by jacobheun** [comment]
@MikeMcl Adding "big.js" to the browser field, https://github.com/MikeMcl/big.js/pull/97, resolves this if you don't want to rollback.

**Comment by MikeMcl** [comment]
I've published *v5.1.2* with `browser` field added and named export restored.
Please let me know if there are any further issues.

**Comment by tripodsan** [comment]
btw, I had the same issue with webpack, even with version 5.2.2. the solution was to change the module resolve order:

```js
      resolve: {
        mainFields: ['main', 'module'],
        extensions: ['.wasm', '.js', '.mjs', '.json'],
      },
```
see: 
- https://webpack.js.org/concepts/module-resolution/#module-paths
- https://webpack.js.org/configuration/resolve/#resolvemainfields
- https://webpack.js.org/configuration/resolve/#resolveextensions
